index.php

```diff
@@ -12,16 +12,21 @@
 $peersResponse = json_decode($values);
 $values = $l->status();
 $statusResponse = json_decode($values);
+$values = $l->search("fossasia");
+$searchResponse = json_decode($values);
 
```

Comment:
We need more tests for search
`
$l->search("fossasia","2015-01-01");
$l->search("fossasia","2015-01-01", "2016-01-01");
$l->search("fossasia","2015-01-01", "2016-01-01", "sudheesh001");
$l->search("fossasia","2015-01-01", "2016-01-01", "sudheesh001", 10);
$l->search("fossasia","", "", "", 10);
$l->search("fossasia","2015-01-01", "", "sudheesh001", 10);
$l->search("fossasia","2015-01-01", "2016-01-01", "", 10);
$l->search("fossasia","", "", "", 10);
```